src/main/java/mate/jdbc/dao/DriverDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverDao {
+    public Driver create(Driver driver);
+
+    public Driver get(Long id);
```

Comment:
Dao should return `Optional`
![image](https://user-images.githubusercontent.com/32339836/227156777-8b281fc4-bf6f-40fd-b141-c7e0d5276493.png)

Reply:
refactored